cernopendata_client/cli.py

```diff
@@ -88,20 +84,24 @@ def get_record(recid, doi, title, output_fields):
 @click.option(
     "--expand/--no-expand", default=True, help="Expand file indexes? [default=yes]"
 )
-def get_file_locations(recid, doi, title, protocol, expand):
+@click.option(
+    "--server",
+    default="http://opendata.cern.ch",
+    type=click.STRING,
+    help="Query server URL",
```

Comment:
The help string can say more explicitly "Which CERN Open Data server to query? [default=http://opendata.cern.ch]".  Please check also how help looks like when running `--help`.